### ISSUE TITLE
Don't run example UI tests on build/release

### ISSUE THREAD

**Issue by PandaMagnus** [IntelliTect/TestTools.Selenate#5]

**Comment by BenjaminMichaelis** [comment]
UI Tests are now passing on windows 🎉

On mac, we are waiting to hear from https://github.com/browser-actions/setup-chrome/issues/329 to see if we can get that going and if it doesn't get resolved, we'll either have to not run the UI tests on the runner or find another way to update chromium... Maybe only run the UI tests on windows and the rest of the tests on both Windows and Mac??

**Comment by PandaMagnus** [comment]
@BenjaminMichaelis I went ahead and just excluded the test runs for now since they're examples. It would probably make most sense to have them be a separate job so that it doesn't block merging or deployment, but will let us know if things drift out of date again.

**Comment by BenjaminMichaelis** [comment]
Yeah we can build another step to do that, should be pretty trivial
